File: src/main/java/logan/pickpocket/commands/MainCommand.java
package logan.pickpocket.commands;

import logan.api.command.BasicCommand;
import logan.api.command.SenderTarget;
import logan.pickpocket.main.PickpocketPlugin;
import org.bukkit.command.CommandSender;

import java.util.Collections;

/**
 * This is the command run when there are no arguments provided and
 * the user is looking for more information about the plugin.
 */
public class MainCommand extends BasicCommand<CommandSender> {

    public MainCommand() {
        super("pickpocket", "pickpocket", 0, 0,
                new String[] { "pp" }, SenderTarget.BOTH, null,
                Collections.emptyList(),
                "Usage:\n/pickpocket (no args)");
    }

    @Override
    public boolean run(CommandSender sender, String[] args, Object data) {
        sender.sendMessage(
                "Pickpocket " + PickpocketPlugin.getInstance().getName() + " " + PickpocketPlugin.getPluginVersion()
                        + "\n" +
                        "/pickpocket toggle - Toggle server-wide pickpocketing.\n" +
                        "/pickpocket status - Check server-wide pickpocketing status.\n" +
                        "/pickpocket trap - Open your trap inventory.\n" +
                        "/pickpocket skills - View your skill levels in a menu.\n" +
                        "/pickpocket setskill <skill name> <level> - Set one of your skill levels.\n" +
                        "/pickpocket reload - Reload the plugin.\n" +
                        "/pickpocket debug giverandom <amount> - Give yourself random items.");
        return true;
    }
}

File: src/main/java/logan/pickpocket/commands/MainCommand.kt
package logan.pickpocket.commands

import logan.api.command.BasicCommand
import logan.api.command.SenderTarget
import logan.pickpocket.main.PickpocketPlugin
import org.bukkit.command.CommandSender

/* This is the command run when there are no arguments provided and
* the user is looking for more information about the plugin. */
class MainCommand : BasicCommand<CommandSender>(
    "pickpocket",
    target = SenderTarget.BOTH,
    permissionNode = "pickpocket.use",
    aliases = arrayOf("pp"),
    usage = """
        Usage:
        /pickpocket (no args)
    """.trimIndent()
) {

    override fun run(sender: CommandSender, args: Array<out String>, data: Any?): Boolean {
        sender.sendMessage(
            """
            Pickpocket ${PickpocketPlugin.instance.name} ${PickpocketPlugin.pluginVersion}
            Admin commands:
            /pickpocket admin notify - Toggle admin notifications for yourself
            /pickpocket admin exempt [name] - Exempt yourself or another player from being stolen from
            /pickpocket admin bypass [name] - Toggle cooldown bypass for yourself or another player
            /pickpocket admin toggle - Toggle pickpocketing on or off.
            /pickpocket admin status - View if pickpocketing is currently enabled or disabled.
    """.trimIndent()
        )
        return true
    }
}